<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hausa Learning App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label{
border: 2px solid gray;
border-radius: 10px;
background-color: beige;
padding:2px;
font-size: 40px;
font-family: Tempus Sans ITC;
}

#pixDisp{
font: 245px Calibri;
color: brown
}

#itemDisp{
font-size: 27px;
font-family: Calibri;
color: brown;
padding:2px;
}

#btn_next{
image: url(:/icons/images/arrow-right.png);
border: none
}

#btn_previous{
image: url(:/icons/images/arrow-left.png);
border: none
}

#btn_next:hover, #btn_previous:hover{
background-color: lightgrey;
}

#btn_next:pressed, #btn_previous:pressed{
background-color: lightslategray;
}

QListWidget{
font-family: calibri;
font-size: 16px;
border: 1px solid lightgray;
padding: 3px
}

#label_example{
padding: 1px;
font: 19px Calibri;
}

#btn_speak{
border:none;
}

#word_1,#word_2,#word_3,#word_4{
border: none
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QLabel" name="pixDisp">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>391</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="hausa_app.qrc">:/icons/images/blank.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>210</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_previous">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="itemDisp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>451</width>
       <height>59</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_example">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>430</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>501</width>
         <height>101</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_chooseAlpha">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>135</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 5px;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c);
padding:2px;
font-family: calibri;
font-size: 13px;
width:90px</string>
         </property>
         <property name="text">
          <string>Choose alphabet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_goBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>135</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 5px;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c);
padding:2px;
font-family: calibri;
font-size: 13px;
width:90px</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUser"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
   <addaction name="actionAdd_new_user"/>
   <addaction name="separator"/>
   <addaction name="actionHistory"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Word"/>
   <addaction name="actionEdit_Word"/>
   <addaction name="actionRemove_word"/>
   <addaction name="separator"/>
   <addaction name="actionLogout"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionView_users"/>
    <addaction name="actionView_admin"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/remove_user.png</normaloff>:/icons/images/remove_user.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Profile</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/edit_user.png</normaloff>:/icons/images/edit_user.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit Profile</string>
   </property>
  </action>
  <action name="actionAdd_Word">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/add-notes.png</normaloff>:/icons/images/add-notes.png</iconset>
   </property>
   <property name="text">
    <string>Add Word</string>
   </property>
   <property name="toolTip">
    <string>Click to add word</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Word">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/edit.png</normaloff>:/icons/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Word</string>
   </property>
   <property name="toolTip">
    <string>Click to edit the word entry</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRemove_word">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/delete-file.png</normaloff>:/icons/images/delete-file.png</iconset>
   </property>
   <property name="text">
    <string>Remove word</string>
   </property>
   <property name="toolTip">
    <string>Click to remove the selected word</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHistory">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/history.png</normaloff>:/icons/images/history.png</iconset>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="toolTip">
    <string>Click to see your view history</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/logout.png</normaloff>:/icons/images/logout.png</iconset>
   </property>
   <property name="text">
    <string>Switch User</string>
   </property>
   <property name="toolTip">
    <string>Click to logout</string>
   </property>
  </action>
  <action name="actionAdd_new_user">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/add_user.png</normaloff>:/icons/images/add_user.png</iconset>
   </property>
   <property name="text">
    <string>Add new user</string>
   </property>
   <property name="toolTip">
    <string>Click to add new admin</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUser">
   <property name="icon">
    <iconset resource="hausa_app.qrc">
     <normaloff>:/icons/images/user.png</normaloff>:/icons/images/user.png</iconset>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionView_users">
   <property name="text">
    <string>View users</string>
   </property>
  </action>
  <action name="actionView_admin">
   <property name="text">
    <string>View administrators</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="hausa_app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
